### DISCUSSION TITLE
ModalService disable OK button?

### DISCUSSION THREAD

**Discussion by michaelcsikos** [ant-design-blazor/ant-design-blazor#2902] [Q&A]
Is it possible to disable the `ModalService` **OK** button, e.g. while the form has validation errors?

**Comment by ElderJames** [comment]
Please try `OkButtonProps`.

![image](https://user-images.githubusercontent.com/7550366/203687409-b378776b-edae-49ea-860d-3e3c0ce2ff73.png)

**Reply by michaelcsikos** [reply]
How do you update the button props when the form is valid?

**Reply by michaelcsikos** [reply]
I've managed to get it to work with my own modal footer.

**Reply by ElderJames** [reply]
That's fine really! Could you please post the solution here? Or a PR is welcome!

**Reply by michaelcsikos** [reply]
My project is using [CSLA](https://github.com/MarimerLLC/csla). I based the custom modal footer on [this example](url).
